lib/email/sender.rb

```diff
@@ -239,6 +242,28 @@ def self.host_for(base_url)
 
     private
 
+    def add_attachments
+      attachment_regex = /<a class="attachment" href="([^"]+)">([^<]*)<\/a>/
```

Comment:
Instead of checking the content of the message, I think it might be better to use the `post_uploads` table.